project-backend/Controllers/JobController.cs

```diff
@@ -43,8 +44,12 @@ public IActionResult GetJobs([FromBody] GetJobsQueryObject query)
                 return Unauthorized(new Error("Not logged in"));
             }
 
+            var test = _jobProvider.QueryJobs().ToArray();
```

Comment:
```suggestion
```
I think this is not intended to go to develop

Reply:
Indeed, it is not :)